styleguide/styles.css

```diff
@@ -923,147 +923,8 @@ pre > code {
   display: block;
   padding: 1rem 1.5rem;
   white-space: pre;
+  display: block;
+  padding: 1em;
+  margin-bottom: 1em;
+  border-bottom: 2px solid black;
 }
-
-/* Tables
-–––––––––––––––––––––––––––––––––––––––––––––––––– */
-th,
```

Comment:
The styles.css file is a compiled file so you should never make changes in this file directly. All of the CSS in this file is compiled from other files in the styleguide